packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -135,49 +157,43 @@ export const disconnectProvider = () => async (dispatch: Dispatch, getState: Get
         type: METADATA.SET_PROVIDER,
         payload: undefined,
     });
+
+    // dispose metadata values (not keys)
     dispatch(disposeMetadata());
 };
 
 const handleProviderError = (error: Error) => (dispatch: Dispatch, getState: GetState) => {
     const { metadata } = getState();
-    const unexpectedError = 'labeling action failed with unexpected error';
-    if (!error) {
-        return dispatch(
-            notificationActions.addToast({
-                type: 'error',
-                error: unexpectedError,
-            }),
-        );
-    }
-    // @ts-ignore todo: type error;
-    switch (error.status) {
-        case 403:
-        case 401:
-            if (getState().metadata.provider) {
-                dispatch(
-                    notificationActions.addToast({
-                        type: 'error',
-                        error: `Failed to sync labeling data with cloud provider (${
-                            metadata!.provider!.type
-                        }). User was logged out.`,
-                    }),
-                );
-                dispatch(disconnectProvider());
-            }
-            break;
-        case 404:
-            dispatch(
-                notificationActions.addToast({
-                    type: 'error',
-                    error: 'Failed to find metadata in cloud provider.',
-                }),
-            );
-            dispatch(disconnectProvider());
-            break;
-        default:
-            dispatch(notificationActions.addToast({ type: 'error', error: unexpectedError }));
-            break;
+    let errorMsg = 'labeling action failed with unexpected error';
```

Comment:
Same comment as above. If the string is user facing, we should localize it.

Reply:
Yeah, this is probably not going to change and we are going to need some "abstract metadata error" so right, I ll add it.